pkg/cmd/server/api/types.go

```diff
@@ -20,6 +20,10 @@ var (
 	DeadKubernetesAPILevels    = []string{"v1beta1", "v1beta2"}
 	DeadOpenShiftAPILevels     = []string{"v1beta1"}
 
+	FeatureAliases = map[string][]string{
```

Comment:
if you don't normalize the alias, you don't get O(1) lookup, you have to iterate. Not a big deal, since the aliases map shouldn't have more than two items in it